tests/PySys/environments/environment_c8y.py

```diff
@@ -69,6 +70,35 @@ def get_thin_edge_device_by_name(self, device_id: str):
                 return device
         return None
 
+    def get_child_device_of_thin_edge_device_by_name(self, thin_edge_device_id: str, child_device_id: str):
+        self.device_fragment = self.get_thin_edge_device_by_name(
+            thin_edge_device_id)
+        internal_id = self.device_fragment['id']
+        child_devices = self.to_json_response(requests.get(
+            url="{}/inventory/managedObjects/{}/childDevices".format(self.c8y_url, internal_id), auth=self.auth))
+        for child_device in child_devices['references']:
+            if child_device_id in child_device['managedObject']['name']:
+                return child_device['managedObject']
+
+        return None
+
+    def get_last_measurements_from_device(self, device_internal_id: str):
+        params = {
+            "source": device_internal_id,
+            "pageSize": 1,
+            "revert": True
+        }
+        res = requests.get(
+            url=self.c8y_url + "/measurement/measurements", params=params, auth=self.auth)
+        measurements_json = self.to_json_response(res)
+        return measurements_json['measurements'][0]
+
+    def delete_managed_object_by_internal_id(self, internal_id: str):
```

Comment:
Is the ID a numeric id or the name of the device? Why does it have to be a string?

Reply:
It's numeric, the managed object ID which Cumulocity gives.